src/main/scala/org/hatdex/hat/api/services/HatClient.scala

```diff
@@ -27,5 +28,5 @@ class HatClient(
       ws: WSClient,
       hatAddress: String) = this(ws, hatAddress, "https://", "v2.6")
   override val host: String = if (hatAddress.isEmpty) "mock" else hatAddress
-  val logger                = play.api.Logger(this.getClass)
+  val logger: Logger        = play.api.Logger(this.getClass)
```

Comment:
As the logger is being changed, it would be good to replace the whole statement with a new approach added in Play 2.7. Line 31 can removed by simply extending class with `Logging`